### ISSUE TITLE
Selectize accessibility issues

### ISSUE THREAD

**Issue by yanivtoledano** [jrief/django-formset#82]
Just came across this library and it's great so far!

Trying the Selectize widget, I have noticed accessibility is off. One way to tell is that navigating the options using the keyboard doesn't work. Pressing the up/down key doesn't move the focused option.

I checked the docs of Tom-Select and can't really tell what is going wrong in django-formset. Maybe someone else has a clue?

**Comment by jrief** [comment]
Well spotted!

Please check my previous commit and tell me if that fix works for you.
